test/lib/ansible_test/_internal/commands/coverage/combine.py

```diff
@@ -137,11 +141,16 @@ def _command_coverage_combine_powershell(args):
     """
     coverage_files = get_powershell_coverage_files()
 
-    def _default_stub_value(lines):
-        val = {}
-        for line in range(lines):
-            val[line] = 0
-        return val
+    if args.stub:
+        find_executable('pwsh', required=True)
+
```

Comment:
This check is redundant with the one performed by `run_command`.

```suggestion
```

Also, checking it here requires `pwsh` even when no PowerShell sources are present.

Reply:
Thanks has been updated.